Objects/unicodeobject.c

```diff
@@ -13408,6 +13546,17 @@ _PyUnicodeWriter_Finish(_PyUnicodeWriter *writer)
     return unicode_result(str);
 }
 
+
+PyObject*
```

Comment:
Nit: inconsistent with the rest of the PR.

```suggestion
PyObject *
```